packages/brand/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./Kaizen/Brand/Brand"
```

Comment:
I know some components have the component living in a `Kaizen` folder, but I don't really know why. `src` makes more sense and seems to be what we're moving towards. Mind changing this?

Reply:
It's because we use the same package for React and Elm code. The consistent directory structure ensures (or ensure**d**) the way Elm modules are imported is consistent. E.g. `import KaizenDraft.ComponentName.ComponentName as ComponentName`

Reply:
It's not needed since I am doubtful this component will have a corresponding Elm component before the Elm code is shuffled out of this repo. 
fyi @mbylstra